js/modules/k6/execution/execution.go

```diff
@@ -90,18 +90,20 @@ func (mi *ModuleInstance) Exports() modules.Exports {
 	return modules.Exports{Default: mi.obj}
 }
 
+var errRunInInitContext = errors.New("getting scenario information outside of vu code is not supported")
```

Comment:
:thinking: I still don't quite like it, but I think this is a bit better:

```suggestion
var errRunInInitContext = errors.New("getting scenario information outside of the VU context is not supported")
```

for sure `VU` should be capitalized

Reply:
:facepalm: